pkg/rid/store/cockroach/identification_service_area_v3.go

```diff
@@ -0,0 +1,204 @@
+package cockroach
```

Comment:
Let's add some documentation to this file saying that its purpose is solely to support backwards compatibility and that it will be deleted from the codebase when all existing production deployments have been upgraded to 3.1.0+.

Reply:
Added